src/neuroconv/schemas/base_metadata_schema.json

```diff
@@ -0,0 +1,171 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "base_metafile.schema.json",
+  "title": "Base schema for the metafile",
+  "description": "Base schema for the metafile",
+  "version": "0.1.0",
+  "type": "object",
+  "required": ["NWBFile"],
+  "properties": {
+    "NWBFile": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "keywords": {
+          "description": "Terms to search over",
+          "type": "array",
+          "items": {
+            "title": "keyword",
+            "type": "string"
+          }
+        },
+        "experiment_description": {
+          "type": "string",
+          "description": "general description of the experiment"
+        },
+        "session_id": {
+          "type": "string",
+          "description": "lab-specific ID for the session"
+        },
+        "experimenter": {
+          "description": "Name of person/people who performed experiment",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "title": "experimenter"
+          }
+        },
+        "identifier": {
+          "type": "string",
+          "description": "A unique text identifier for the file. If one is not provided it will be randomly assigned"
+        },
+        "institution": {
+          "type": "string",
+          "description": "Institution(s) where experiment is performed"
+        },
+        "lab": {
+          "type": "string",
+          "description": "Lab where experiment was performed"
+        },
```

Comment:
@garrettmflynn Though this PR is mostly a NeuroConv thing, this part can factor into the GUIDE as well

In particular, I recall that the institution and lab are things that we extract early on in a separate manner - nonetheless, this information should then copied over to each metadata form for each session propagating from that data set; mostly because, like the publication DOI, these are not things that change over sessions or even experiments

Reply:
Yes. they should be filled as default values which the user can then change session-by-session if they want